src/pages/Profile/index.jsx

```diff
@@ -19,16 +18,57 @@ import {
 } from "../../js/sounds";
 import { useLocation, useNavigate } from "react-router-dom";
 
+import { AnimatedButton } from "../../reusableComponents";
 import ErrorBoundary from "../../reusableComponents/ErrorBoundary";
 import { FaUserCircle } from "react-icons/fa";
+import LogoutIcon from "@mui/icons-material/Logout";
+import Modal from "@mui/material/Modal";
+import SettingsIcon from "@mui/icons-material/Settings";
+// import Modal from "@mui/material/Modal";
 import ViewsCounter from "./views";
 import firebase from "firebase/compat/app";
+import logo from "../../assets/logo.webp";
+import { makeStyles } from "@mui/styles";
 import { useSnackbar } from "notistack";
 
 const Post = lazy(() => import("../../components/Post"));
 const SideBar = lazy(() => import("../../components/SideBar"));
 
+export function getModalStyle() {
```

Comment:
why were you adding this function again if you could just import it?

Reply:
I removed it from app.js and then pasted it to the profile page. Later I realized that it is getting used somewhere else also. So forgot to remove it from Profile.js.